src/main/java/org/myrobotlab/service/Servo.java

```diff
@@ -1631,6 +1631,12 @@ public void setSpeed(Double speed) {
     // TODO Auto-generated method stub
     
   }
+
+  @Override
+  public List<String> refreshEncoders() {

```

Comment:
This method should not be here.  If we want a way to get all "encoders" we should ask Runtime for them.

Reply:
I'd be ok with removing it ...   same **should** go for refreshControllers too which has been part of ServoControl  .. if encoder goes controllers should go  (means some UI work at least...)